hooks/useChartDimensions.js

```diff
@@ -53,9 +53,9 @@ export const useChartDimensions = (passedSettings) => {
         changeHeight(entry.contentRect.height);
     });
 
-    resizeObserver.observe(element);
+    element !== undefined && resizeObserver.observe(element);
```

Comment:
nit: 
While both work, can we have this as 

```
if(!isUndefined(element)){
    resizeObserver.observe(element);
}
```

for better readability, and because it's easier to debug and look for side effects.